assets/js/modules/analytics-4/components/widgets/TopRecentTrendingPages.js

```diff
@@ -0,0 +1,208 @@
+/**
+ * TopCitiesWidget component.
```

Comment:
This title doesn't match the filename/component name, by that raises a separate question: why rename this anyway? With this new filename it's the only KMW in that folder I can see whose component name/filename _doesn't_ end with `Widget`. I don't know what the motivation for that is, as I didn't see it in the ACs of the issue and it was mentioned in the IB but I'm not sure why.

cc @nfmohit who wrote the IB, maybe you can shed some light on the renaming here? 😄

Reply:
Hi @tofumatt. The naming in the IB was an oversight. Since all widget names end with `Widget`, we should also do the same here. Thanks for checking!